images/router/haproxy/conf/haproxy-config.template

```diff
@@ -239,7 +239,11 @@ backend be_tcp_{{$cfgIdx}}
 {{ if ne (env "ROUTER_SYSLOG_ADDRESS" "") ""}}
   option tcplog
 {{ end }}
+{{ if eq (env "ROUTER_DISABLE_STICKY_SESSIONS" "") "" }}
```

Comment:
It would be admin provided, so it's safe in the sense that an admin who provides a broken value can always roll back.